src/facade/lang.ts

```diff
@@ -162,7 +162,7 @@ export function isJsObject( o: any ): boolean {
 
 export function isArguments(value: any): boolean {
   // Safari 8.1 incorrectly makes `arguments.callee` enumerable in strict mode.
-  return ('lenght' in value) && Object.prototype.hasOwnProperty.call(value, 'callee') &&
+  return ('length' in value) && Object.prototype.hasOwnProperty.call(value, 'callee') &&
```

Comment:
This typo fix was not explicitly needed as part of the feature work, but I lucked into finding and didn't want to leave it!